isisd/isisd.h

```diff
@@ -128,6 +128,9 @@ struct isis_area {
 	 */
 	int lsp_regenerate_pending[ISIS_LEVELS];
 
+	int bfd_signalled_down;
+	int bfd_force_spf_refresh;
```

Comment:
it would be better to use booleans for these two variables to improve code readability

Reply:
I chose int's bc they were used that way in the ISIS code. Should I still change this?

Reply:
I would -- and potentially open an issue to clean the int's up in the IS-IS code... In the long run, we shouldn't be using int's as boolean for readability...

Reply:
Personally I can't really cope with having different styles in the same document. @rzalamena what do you think?
Opening an issue is certainly a good idea!

Reply:
I would just replace it with `bool` if it is trivial. I'm not so sure about the issue, because it might just end up forgotten in the pile of other issues.

Maybe we should start a ToDo list of trivial things like this (e.g. project cards), it seems there are people who like doing this kind of code changes.

Reply:
I changed to booleans :)